modules/_node-scripts/bundle/css/processCSSFiles.mjs

```diff
@@ -0,0 +1,63 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import fs from 'fs/promises';
+import {r2} from 'liferay-theme-tasks';
+import path from 'path';
+
+import {BUILD_RESOURCES_PATH, SRC_PATH} from '../../util/constants.mjs';
+import expandGlobs from '../../util/expandGlobs.mjs';
```

Comment:
Can we use 'fast-glob' instead of this? I didn't realize we added this util.

it was always a pain to work with in npm-scripts...

Reply:
unless you think this is better for some reason, then we can remove fast-glob

Reply:
Nope, I used it because it was there :shrug: 

I know in the past we tried not to use too many dependencies for security and configuration management reasons but I think in this case it's OK to have an external dep.